plugins/modules/nsupdate.py

```diff
@@ -253,6 +253,9 @@ def __init__(self, module):
         self.dns_rc = 0
 
     def txt_helper(self, entry):
+        if len(entry) > 255:
+            entry = [entry[i:i+255] for i in range(0, len(entry), 255)]
+            entry = '" "'.join(entry)
```

Comment:
This causes a new bug: if the user already provided a correctly quoted and splitted TXT record, this can change the record's value by inserting `" "` at the wrong position (for example assume that `" "` is inserted in the middle of `\"`: then you get `\" ""`, which does something else than `" "\"` or `\"" "` which you get by inserting `" "` at the beginning of the end of `\"`).

Reply:
Good catch. Thanks.